app/src/test/kotlin/me/myshows/android/api/jsonrpc/JsonRPCTest.kt

```diff
@@ -0,0 +1,81 @@
+package me.myshows.android.api.jsonrpc
+
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import me.myshows.android.api.BaseMockWebServerTest
+import okhttp3.mockwebserver.MockResponse
+import okhttp3.mockwebserver.RecordedRequest
+import okio.Buffer
+import org.assertj.core.api.Java6Assertions.assertThat
+import org.junit.Before
+import org.junit.Test
+import retrofit2.Call
+import retrofit2.Retrofit
+import retrofit2.http.Body
+import retrofit2.http.POST
+
+class JsonRPCTest : BaseMockWebServerTest() {
+
+    private val mapper: ObjectMapper = jacksonObjectMapper()
+    private lateinit var api: TestApi
+
+    @Before
+    fun setup2() {
```

Comment:
Maybe rename this method to `initialize`, since name `setup` is used in `BaseMockWebServerTest`, or 
```
private val api = Retrofit.Builder()
            .addConverterFactory(JsonRPCConverterFactory.create())
            .baseUrl(resolve("/"))
            .build()
            .create(TestApi::class.java)
```

Reply:
Second variant will break all, because it uses `resolve("/")` method and in your case it will be called before `setup` of `BaseMockWebServerTest` and call `MockWebServer.start()`. And when we will call `MockWebServer.start()` the second time, we get exception